TM1py/Services/HierarchyService.py

```diff
@@ -211,6 +211,30 @@ def remove_all_edges(self, dimension_name: str, hierarchy_name: str = None, **kw
         }
         return self._rest.PATCH(url=url, data=json.dumps(body), **kwargs)
 
+    def remove_edges_under_consolidation(self, dimension_name: str,  hierarchy_name: str,
+                                         consolidation_element: str, **kwargs) -> Response:
+        """
+        :param dimension_name: Name of the dimension
+        :param hierarchy_name: Nmae of the hierarchy
+        :param consolidation_element: Name of the Consolidated element
+        :return: response
+        """
+        hierarchy = self.get(dimension_name, hierarchy_name)
+        from TM1py.Services import ElementService
+        element_service = ElementService(self._rest)
+        elements_under_consolidations = element_service.get_members_under_consolidation(dimension_name, hierarchy_name,
+                                                                                        consolidation_element)
+        elements_under_consolidations.append(consolidation_element)
+        remove_edges = []
+        for element, _ in hierarchy.edges.items():
```

Comment:
I think it's better to write 
`for edge in hierarchy.edges:`
here. Or:
I think it's better to write 
`for (parent, component) in hierarchy.edges:`
here.

Reply:
nice..